app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/SettingsActivity.kt

```diff
@@ -0,0 +1,123 @@
+package nerd.tuxmobil.fahrplan.congress.settings
+
+import android.annotation.TargetApi
+import android.content.Context
+import android.content.Intent
+import android.graphics.drawable.ColorDrawable
+import android.os.Build
+import android.os.Bundle
+import android.preference.Preference
+import android.preference.Preference.OnPreferenceChangeListener
+import android.preference.Preference.OnPreferenceClickListener
+import android.preference.PreferenceCategory
+import android.preference.PreferenceFragment
+import android.preference.PreferenceScreen
+import android.provider.Settings
+import androidx.appcompat.widget.Toolbar
+import androidx.core.content.ContextCompat
+import androidx.core.os.bundleOf
+import nerd.tuxmobil.fahrplan.congress.BuildConfig
+import nerd.tuxmobil.fahrplan.congress.MyApp
+import nerd.tuxmobil.fahrplan.congress.R
+import nerd.tuxmobil.fahrplan.congress.alarms.AlarmServices
+import nerd.tuxmobil.fahrplan.congress.base.BaseActivity
+import nerd.tuxmobil.fahrplan.congress.contract.BundleKeys
+import nerd.tuxmobil.fahrplan.congress.extensions.getAlarmManager
+import nerd.tuxmobil.fahrplan.congress.extensions.toSpanned
+import nerd.tuxmobil.fahrplan.congress.utils.FahrplanMisc
+
+class SettingsActivity : BaseActivity(R.layout.settings) {
+
+    companion object {
+
+        private const val LOG_TAG = "SettingsActivity"
+
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val toolbar = findViewById<Toolbar>(R.id.toolbar)
+        setSupportActionBar(toolbar)
+        val actionBarColor = ContextCompat.getColor(this, R.color.colorActionBar)
+        supportActionBar!!.setBackgroundDrawable(ColorDrawable(actionBarColor))
+
+        if (savedInstanceState == null) {
+            fragmentManager.beginTransaction()
+                    .add(R.id.container, SettingsFragment())
+                    .commit()
+            MyApp.LogDebug(LOG_TAG, "onCreate fragment created")
+        }
+    }
+
+    class SettingsFragment : PreferenceFragment() {
```

Comment:
I'd move the fragment to a separate file.

Reply:
Done in 0035e5eac9984ae1eb073478ad87593f61e65dc3.

Reply:
Done in 0035e5eac9984ae1eb073478ad87593f61e65dc3.